homework-submission/myapp4/src/App.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import './App.css';
+import Header from './Components/Header';
+import TodosCounter from "./Components/TodosCounter";
+import { inject, observer } from 'mobx-react';
+import Form from './Components/Form';
+
+
+@inject("TodosStore")
+@observer
+class App extends Component {
+  
+  deleteTodo = id => {
+    this.props.TodosStore.removeTodo(id);
+  };
+
+  isTaskDone = todo => {
+    this.props.TodosStore.isDone(todo);
+  };
+
+  renderTodos = () => {
+    if (this.props.TodosStore.Todos.length === 0)
+      return <p className="noItems">No items...</p>;
+
+    const textStyle = {
+      textDecoration: "line-through",
+      backgroundColor: "#0455d8"
+    };
+
+    return (
+      <ul>
+        {this.props.TodosStore.Todos.map((todo, id) => {
```

Comment:
While you have an `id` in your `todo` object, then what is `id` here? can it be instead?